Makefile

```diff
@@ -1,5 +1,5 @@
 BINARY=terraform-provider-tozny
-VERSION=0.9.81
+VERSION=0.10.0
```

Comment:
I usually do a find and replace across all *.tf files as well to auto update the examples to reference the latest version as well, but I can do that when its time to officially publish a new version